doc/src/manual/dates.md

```diff
@@ -181,7 +184,7 @@ julia> dt2 = DateTime(2000,2,1)
 2000-02-01T00:00:00
 
 julia> dt - dt2
-381110402000 milliseconds
+381110400000 milliseconds
```

Comment:
My guess is the original result came from when Dates was accounting for leap seconds by default (back before it was in Base).